tGUI.m

```diff
@@ -1,33 +1,33 @@
 %
-% tGUI(EXPERIMENT_NAME_STRING)
+% tGUI;
```

Comment:
Please revert back to the original function name (ln 2) with the text "EXPERIMENT_NAME_STRING" in the function call.